enqueue/fs/0.3/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/",
+        "var/": "%VAR_DIR%"
```

Comment:
Is the `var/queue/.gitkeep` really needed? The package should be able to create it when it does not exist, right?